api/serializers.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import serializers
+
+from history.models import HistoryEntry
+
+__all__ = [
+    "HistoryEntryListSerializer",
+]
+
+
+class HistoryEntryListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = HistoryEntry
+        fields = [
+            "message",
+            "created",
+        ]
```

Comment:
@SebCorbin I removed the object's uuid from here, it's not needed as we already know it :)